src/chat/menu/controller.js

```diff
@@ -24,4 +50,29 @@ module.exports = class MenuController {
             }
         }
     }
+
+    myProfile() {
+        this.cardService.profileScreen = true;
+        this.cardService.archiveScreen = false;
+        this.cardService.uptodateScreen = false;
+        this.cardService.searchScreen = false;
+        this.cardService.recycleBinScreen = false;
+        this.cardService.userScreen = false;
+        this.uptodateScreen = false;
+        this.cardService.trigger('UptodateScreen');
+        this.cardService.trigger('Profile');
+    }
+
+    shoppingList() {
+        this.uptodateScreen = true;
+        this.profileScreen = false;
+        this.cardService.profileScreen = false;
+        this.cardService.archiveScreen = false;
+        this.cardService.uptodateScreen = true;

```

Comment:
We have discussed each other today and we will chance screen structure. But we have to discuss figure out how to handle this.